### DISCUSSION TITLE
Unguided Exercise - HTML - Tables

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/mern#231] [Tasks]
### Table-1
![table-1](https://github.com/akash-coded/mern/assets/60097560/344c2844-82d8-41f0-b0b2-30d984eeb553)

### Table-2
[table-2.zip](https://github.com/akash-coded/mern/files/13403850/table-2.zip)

### Table-3
[table-3.zip](https://github.com/akash-coded/mern/files/13403852/table-3.zip)

### Table-4
![table-4](https://github.com/akash-coded/mern/assets/60097560/5e28c46c-3d40-47d7-8bc2-ae88e1723be2)

**Comment by bacchus19408** [comment]
table #2
<img width="457" alt="Screen Shot 2023-11-19 at 11 51 43 AM" src="https://github.com/akash-coded/mern/assets/4041048/5a5c219c-edf0-446b-b87a-c377e930dabb">

**Comment by priyeranjan** [comment]
<img width="952" alt="image" src="https://github.com/akash-coded/mern/assets/7481567/7fccf362-25e8-4eed-9bb1-b6b958469d49">

**Comment by priyeranjan** [comment]
<img width="955" alt="image" src="https://github.com/akash-coded/mern/assets/7481567/55424fb2-f563-4a01-a58d-16cf5066808c">

**Comment by priyeranjan** [comment]
<img width="953" alt="image" src="https://github.com/akash-coded/mern/assets/7481567/0fa0c251-f8d9-4cbd-ab46-60c3a8ce7670">

**Comment by Dhanalakshmi-S-23** [comment]
![Image3_output](https://github.com/akash-coded/mern/assets/151384783/54bb63b6-bbed-496d-b283-7df0e1248918)

**Comment by bacchus19408** [comment]
table - 4
<img width="675" alt="Screen Shot 2023-11-19 at 12 15 12 PM" src="https://github.com/akash-coded/mern/assets/4041048/9cdac5e5-a839-4245-8078-b72a395d7c8d">

**Comment by Balajik87-hub** [comment]
<img width="1440" alt="image" src="https://github.com/akash-coded/mern/assets/64579087/fe3621ae-feeb-4bb3-bd56-30cc834c3253">

**Comment by rosekthomas** [comment]
<img width="618" alt="Screenshot 2023-11-23 at 4 54 02 PM" src="https://github.com/akash-coded/mern/assets/8777388/9b226186-4cb9-4362-b870-7a27c50ef9d9">

![table4](https://github.com/akash-coded/mern/assets/8777388/3f382eb1-4937-47ca-87d2-c107083f0e12)

![t3](https://github.com/akash-coded/mern/assets/8777388/c8c04702-2ed0-431b-aca8-0dc798d2695d)

**Comment by priyeranjan** [comment]
<img width="960" alt="image" src="https://github.com/akash-coded/mern/assets/7481567/c07c62e8-6e31-49df-9a77-c0a84adb5fd7">

**Comment by Dipalivrma** [comment]
![image](https://github.com/akash-coded/mern/assets/119152780/269948d6-f580-4229-8bcc-1af328b72fca)

**Comment by clayp2023** [comment]
![image](https://github.com/akash-coded/mern/assets/46467334/fe06e612-071d-4bab-9354-1074b595c614)

![image](https://github.com/akash-coded/mern/assets/46467334/13d228c4-c80f-49c5-b970-273eef9b4550)


![image](https://github.com/akash-coded/mern/assets/46467334/b95ad668-14bf-49ab-8014-63c2f7c9115b)


![image](https://github.com/akash-coded/mern/assets/46467334/03aa13b5-ad11-4c7b-a694-b04300fd8fed)

**Comment by Dhanalakshmi-S-23** [comment]
![Image4_output](https://github.com/akash-coded/mern/assets/151384783/f1d5e119-0a33-415b-bf54-8083a8abba57)

**Comment by arvid-mvo** [comment]
![table_1_submit](https://github.com/akash-coded/mern/assets/101291172/763e9441-dae1-4842-8add-6bd3749ce75f)
![table_2_submit](https://github.com/akash-coded/mern/assets/101291172/dd1d68db-4abd-4955-a288-e1f34aee0c5b)
![table_3_submit](https://github.com/akash-coded/mern/assets/101291172/4e5a4876-9df4-4f96-a1fc-e4d3c16425ae)
![table_4_submit](https://github.com/akash-coded/mern/assets/101291172/9c3a179a-693b-4818-b72d-52e0419adffd)

**Comment by bacchus19408** [comment]
table-3 with css background and hyperlinks:
<img width="553" alt="Screen Shot 2023-11-19 at 1 50 12 PM" src="https://github.com/akash-coded/mern/assets/4041048/50b8fcef-84dc-4ad9-a27a-7dc7cd833d28">

**Comment by Dhanalakshmi-S-23** [comment]
![Image1_output](https://github.com/akash-coded/mern/assets/151384783/e528316f-b5bb-4ba7-a6c0-fa505cebf291)

**Comment by vpervela** [comment]
Table 1
<img width="414" alt="Screenshot 2023-11-19 at 3 16 15 PM" src="https://github.com/akash-coded/mern/assets/101532142/b124aaea-19e2-4d57-b2f1-8d30b64dc851">

Table 4
<img width="881" alt="Screenshot 2023-11-19 at 4 22 42 PM" src="https://github.com/akash-coded/mern/assets/101532142/016b7912-42c7-4707-b8d4-cab48bc4f114">

Table3
<img width="881" alt="Screenshot 2023-11-19 at 4 52 13 PM" src="https://github.com/akash-coded/mern/assets/101532142/4be3b8e6-7221-4e30-990f-3756911435fc">

**Comment by Dhanalakshmi-S-23** [comment]
![Image2_output](https://github.com/akash-coded/mern/assets/151384783/43630b81-82c9-43cb-b1bf-75ef959a7aa2)

**Reply by Sunil-nani** [reply]
hello madam please send me the code once

**Comment by ujjwal432** [comment]
<img width="1084" alt="Screenshot 2023-11-20 185224" src="https://github.com/akash-coded/mern/assets/61551049/0dda3062-4795-43c2-b36c-0a5996ce1be8">
<img width="1096" alt="Screenshot 2023-11-20 194710" src="https://github.com/akash-coded/mern/assets/61551049/55870378-010b-4a95-9b8a-9193279621d3">

**Comment by SwetaSaha1** [comment]
![Screenshot 2023-11-20 at 8 38 36 PM](https://github.com/akash-coded/mern/assets/127398967/e4c09e00-8185-4e96-bf1e-cc154dbe8eb8)
![Screenshot 2023-11-20 at 8 39 03 PM](https://github.com/akash-coded/mern/assets/127398967/0c5c7569-905c-4834-9531-db3565385075)
![Screenshot 2023-11-20 at 8 39 30 PM](https://github.com/akash-coded/mern/assets/127398967/99f0506e-b345-456e-95cc-d071b43bdc4f)
![Screenshot 2023-11-20 at 8 40 01 PM](https://github.com/akash-coded/mern/assets/127398967/7e7c584f-446b-4eeb-af01-1a7d49ca7568)

**Comment by Sunil-nani** [comment]
TABLE 1
![Screenshot (5)](https://github.com/akash-coded/mern/assets/144610117/7c635738-7514-4c43-b575-3f85616e4060)

**Comment by padma1998lela** [comment]
![image](https://github.com/akash-coded/mern/assets/62737217/850837df-bca2-4488-88c5-38ce9d48fe42)

**Comment by Sunil-nani** [comment]
TABLE 3
![Screenshot (6)](https://github.com/akash-coded/mern/assets/144610117/6af46770-6d99-456c-8c38-7111fdf904d5)

**Comment by Umarani1711** [comment]
<img width="455" alt="Table1" src="https://github.com/akash-coded/mern/assets/78922312/5a38e1f0-d727-4361-bf4f-12d6c2f66c19">
<img width="431" alt="table2" src="https://github.com/akash-coded/mern/assets/78922312/af7422f4-dce6-4015-b3f1-86b4e74b1142">
<img width="249" alt="table3" src="https://github.com/akash-coded/mern/assets/78922312/4ddd8463-ae44-46bf-8a2d-6f5d4ea07515">
<img width="644" alt="table4" src="https://github.com/akash-coded/mern/assets/78922312/60fa2500-d122-44ea-8f9a-9bd1a4064233">

**Comment by padma1998lela** [comment]
![Capture](https://github.com/akash-coded/mern/assets/62737217/b715f7ab-2fca-4d49-9d67-6057bde2a774)

**Comment by padma1998lela** [comment]
![image](https://github.com/akash-coded/mern/assets/62737217/7ee14c1e-a1e5-451b-9894-998ab93564ea)

**Comment by AmanBharti07** [comment]
![image](https://github.com/akash-coded/mern/assets/143952663/9419c5e1-8912-4b58-b13d-1f44809384f5)

**Comment by padma1998lela** [comment]
![image](https://github.com/akash-coded/mern/assets/62737217/abcd0062-aa2d-40a8-a3ac-da6d528ed2c5)

**Comment by AmanBharti07** [comment]
![image](https://github.com/akash-coded/mern/assets/143952663/8b81fd16-3717-4c58-aaaa-a97b745b1b56)

**Comment by Sunil-nani** [comment]
TABLE 4
![Screenshot (8)](https://github.com/akash-coded/mern/assets/144610117/261a52bc-d212-48e8-8134-7d084d4eab70)

**Comment by ghost** [comment]
![Screenshot (263)](https://github.com/akash-coded/mern/assets/135525365/b51750d8-c485-41f7-8571-08943dee0d9e)
 Table-3

**Comment by ghost** [comment]
Table -1

![Screenshot (264)](https://github.com/akash-coded/mern/assets/135525365/06e78d20-ca59-44c2-b4fc-24bab4f7e406)

**Comment by ghost** [comment]
Table-4

![Screenshot (265)](https://github.com/akash-coded/mern/assets/135525365/a4cecaaf-14e4-4e1e-a7ba-a543fb6357fb)

**Comment by AmanBharti07** [comment]
![image](https://github.com/akash-coded/mern/assets/143952663/d60355b3-8c05-4d37-8c62-5ecd5d756a6f)

**Comment by Lellagopi143** [comment]
![Screenshot 2023-11-21 194702](https://github.com/akash-coded/mern/assets/151382935/383f0f45-7601-4d84-bae0-1f283dc0a047)
![Screenshot 2023-11-22 144625](https://github.com/akash-coded/mern/assets/151382935/6ac352b4-a57c-4210-b0b4-7e1bd21b3b6d)
![Screenshot 2023-11-21 213133](https://github.com/akash-coded/mern/assets/151382935/5de2b192-ebd1-4400-9ef0-b89d45c6a264)
![Screenshot 2023-11-21 203700](https://github.com/akash-coded/mern/assets/151382935/62ffbd1f-0952-4b0e-b3cc-597e084bd9db)

**Comment by AbiramiMadhusudan28** [comment]
<img width="1470" alt="Screenshot 2023-11-24 at 10 36 04 PM" src="https://github.com/akash-coded/mern/assets/151383196/16060415-81e0-42d4-9fd9-b0badcdaf3c0">

**Comment by AbiramiMadhusudan28** [comment]
<img width="1470" alt="Screenshot 2023-11-24 at 10 37 32 PM" src="https://github.com/akash-coded/mern/assets/151383196/557e5da0-7b16-47e6-83f9-97e87604ba71">

<img width="1470" alt="Screenshot 2023-11-24 at 10 38 14 PM" src="https://github.com/akash-coded/mern/assets/151383196/f02a48c3-b962-44f5-9afe-adc9631ff8b3">

**Comment by Geethanjali5** [comment]
<img width="906" alt="Screenshot 2023-11-23 at 1 03 23 AM" src="https://github.com/akash-coded/mern/assets/147892641/56b2060f-bcde-4908-a0bd-78aba7c75b3a">

**Comment by Dipalivrma** [comment]
![image](https://github.com/akash-coded/mern/assets/119152780/cb5f8a51-c71e-4257-a3d2-f1c58b8f1640)

**Comment by Dipalivrma** [comment]
![image](https://github.com/akash-coded/mern/assets/119152780/2a674a19-0413-4b9e-ba33-2f60e49bd711)

**Comment by monalip92** [comment]
![image](https://github.com/akash-coded/mern/assets/144552652/569ba70b-29ac-44d0-a4fc-4a2ab105b4c6)

**Comment by SimplySneha27** [comment]
![image](https://github.com/akash-coded/mern/assets/151423861/4d4d9b67-509c-49b0-ad13-70d913f24410)

**Comment by monalip92** [comment]
![image](https://github.com/akash-coded/mern/assets/144552652/114cabde-9b1c-4e54-a462-cad44c51d1ec)

**Comment by PUMAGrisen** [comment]
![html-time-table](https://github.com/akash-coded/mern/assets/151846056/2e74961b-9ba8-4e23-951d-935a4898901a)

**Comment by SimplySneha27** [comment]
Table 2
![image](https://github.com/akash-coded/mern/assets/151423861/2976bba7-5f4d-4c6d-93f3-923efe3934a9)

Table 3
![image](https://github.com/akash-coded/mern/assets/151423861/db146d05-db79-492e-ac42-79eaf6b0b923)

Table 4
![image](https://github.com/akash-coded/mern/assets/151423861/a0240caf-7b30-4db2-9ab4-99327e1eb25e)

**Comment by VIJAY1999v** [comment]
![table 1](https://github.com/akash-coded/mern/assets/126997615/84bd6afb-9d90-4a7a-bb00-ed90f44cf507)

**Comment by VIJAY1999v** [comment]
![table2](https://github.com/akash-coded/mern/assets/126997615/539f5df9-cc25-4662-a14c-d70f8d5d8de6)

**Comment by VIJAY1999v** [comment]
![table 3](https://github.com/akash-coded/mern/assets/126997615/1cd27824-4e5e-47ca-b82a-7e7e3ab75a38)
